NMS-14619: Quick Start: Determine service availability (#5350)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/downtime-model.adoc

[[ga-service-assurance-downtime-model]]
= Detect Short Service Outages

By default the monitoring interval for a service is five minutes.

> To detect short service outages, caused, for example, by automatic network rerouting, use the downtime model.

Reviewer: ```suggestion
To detect short service outages, like those caused by automatic network rerouting, use the downtime model.
```

On a detected-service outage, the interval is reduced to every 30 seconds for five minutes.
If the service comes back within five minutes, a shorter outage is documented and the impact on service availability can be less than five minutes.

> This configurable behavior is called "downtime model".

Reviewer: ```suggestion
This configurable behavior is called the "downtime model."
```

> fix this: downtime model defines ...

Reviewer 2: ?
